playbooks/roles/ocp-lso/tasks/main.yaml

```diff
@@ -0,0 +1,245 @@
+---
+# tasks file for playbooks/roles/ocp-lso
+
+# Check the health of Cluster Operators
+- name: Check if cluster operators and nodes are healthy
+  include_role:
+    name: check-cluster-health
+
+# Checking if worker node contains 500 Gb volume attached
+- name: Check if 500GB volume is attached to worker nodes
+  shell: ssh -o StrictHostKeyChecking=no core@worker-0 lsblk | grep 500
+  register: volume_check
+  failed_when: volume_check.rc != 0  # Fail if the volume is not found
+  changed_when: false  # This task doesn't change anything
+  ignore_errors: false  # Do not ignore errors
```

Comment:
ignore_erros is not necessary here or it should be true

Reply:
removed